e2e/config/config.go

```diff
@@ -28,12 +28,19 @@ type Cluster struct {
 	KubeconfigPath string
 }
 
+type Test struct {
+	Workload string
+	Deployer string
+	PVCSpec  string
+}
+
 type Config struct {
 	// User configurable values.
 	ChannelNamespace string
 	GitURL           string
 	Clusters         map[string]Cluster
 	PVCSpecs         []PVCSpec
```

Comment:
This is the on disk format, but we really want:

    PVCSpecs map[string]PVCSpec

So we can validate test's PVCSpec property.

Reply:
Leaving this for later.